jck/compiler.api/playlist.xml

```diff
@@ -129,7 +129,7 @@
 		<variations>
 			<variation>NoOptions</variation>
 		</variations>
-		<command>$(GEN_JTB) tests=api/java_rmi testsuite=COMPILER'; \
+		<command>$(GEN_JTB) tests=api/java_rmi testsuite=COMPILER$(USEQ); \
```

Comment:
Instead of creating a new variable, can use use `$(Q)`?

Reply:
We can not, since that extra quote needs to be used only for the two-node case.